src/app/component/crossfade-buttons/crossfade-buttons.component.html

```diff
@@ -1,4 +1,5 @@
-<button (click)="func(1)">1</button>
-<button (click)="func(5)">5</button>
-<button (click)="func(10)">10</button>
-<button (click)="func(30)">30</button>
+<button *ngFor="let duration of durations" (click)="click(duration)"
+        class="btn" [ngClass]="isSelected(duration) ? 'btn-selected' : 'btn'">
```

Comment:
remind me what `?` does in this context again?

Reply:
let x;
if (bool) {
  x = 'yay';
} else {
  x = 'nay';
}

Is equivalent to

x = bool ? 'yay' : 'nay'

It's called the ternary operator